vcf2fhir/converter.py

```diff
@@ -70,6 +70,9 @@ class Converter(object):
     (or 0-start, half-open) whereas VCF files and FHIR output are 1-based (or 1-start,
     fully-closed).
 
+    **ratio_ad_dp** (optional)(default value = 0.99): This ratio determine whether to assign Homoplasmic or Heteroplasmic
+     If allelic depth (FORMAT.AD) / read depth (FORMAT.DP) is greater than ratio_ad_dp then allelic state is
+      homoplasmic; else heteroplasmic.

```

Comment:
@rhdolin  does this description looks good to you ? Or there is some bioinformatics way to explain this.

Reply:
Thanks for this. I'll update manual page with revised text.

Reply:
I am sorry , @srgothi92 @rhdolin , I didn't have the full knowledge as to what that parameter actually represented/meant so I just used the issue's explanation to describe it .